src/containers/SelectionBar.jsx

```diff
@@ -24,15 +24,15 @@ const SelectionBar = ({ t, selected, selectedCount, album, onHide, onAddToAlbum,
     <span className={styles['coz-selectionbar-separator']} />
     <button
       disabled={selectedCount === 0}
-      className={styles['pho-action-album-add']}
+      className={styles['coz-action-album-add']}
```

Comment:
Is it wise to move albums-related styles to cozy-ui? These styles look app-specific to me.